### ISSUE TITLE
Bad Contrast for keywords in Dark Solarized Theme

### ISSUE THREAD

**Issue by prnadago** [justcla/VSColorThemes#4]
Discovered an issue with the dark solarized theme. Most of the keywords seem to be black, which really are hard to see with a dark theme. https://imgur.com/tus3o60

**Comment by HighwayStar54** [comment]
Yep, thanks for saving me the time to write that, I got redirected here from the Q&A section of this extension.

**Comment by prnadago** [comment]
Now being tracked here - https://github.com/Microsoft/VS-ColorThemes/issues/3
